.github/workflows/publish.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       if: matrix.os == 'ubuntu-18.04'
       uses: actions/upload-artifact@master
       with:
-        name: titan-cli-0.3.0-linux_amd64.zip
```

Comment:
Should we be taking the version from the tag instead of hard-coding "0.3.0" here?

Reply:
Good catch. Let me look into this

Reply:
I think this is only for the artifact upload, the actual artifact name comes from the script which is based on the version. I'll remove this number though.